packages/bastet_async/bastet_async.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Async implementations for bastet"
+maintainer: ["me@risto.codes"]
+authors: ["Risto Stevcev"]
+license: "BSD-3-Clause"
+tags: ["category theory" "abstract algebra" "algebra" "cats" "async"]
+homepage: "https://github.com/Risto-Stevcev/bastet-async"
+doc: "https://risto-stevcev.github.io/bastet-async"
+bug-reports: "https://github.com/Risto-Stevcev/bastet-async/issues"
+depends: [
+  "ocaml" {>= "4.06.1"}
+  "bastet" {>= "1.2.4"}
+  "async_kernel" {>= "v0.12.0"}
+  "async_unix" {>= "v0.12.0" & with-test}
+  "core" {>= "v0.12.0" & with-test}
+  "alcotest" {>= "1.0.1" & with-test}
+  "alcotest-async" {>= "1.0.1" & with-test}
+  "merlin" {>= "3.3.3" & with-test}
+  "ocamlformat" {>= "0.13.0" & with-test}
```

Comment:
I apparently missed that in your previous PRs for `bastet` but I'm a bit surprised to see `merlin` and `ocamlformat` as test dependencies. I'm curious, how do you use them during tests?

Reply:
It's just a convenience because I recreate the local switch to test before deploying

Reply:
This not really the purpose of test dependencies, also this contribute to clogging up our CI. Would it be possible to have a separate opam file for that purpose if you really need it? See https://github.com/ocaml/opam-repository/pull/15964#issuecomment-596545221 for a similar answer I gave a few weeks ago.

Reply:
Yeah, I think it might be better keeping this in the `Makefile` anyway -- I still have to run multiple commands with this in place anyway, but if I put it in `make` I can set it up to do all of this stuff in one command.